src/components/include/DefaultImportModalDialog.jsx

```diff
@@ -0,0 +1,323 @@
+import nativeFormats from "@exabyte-io/made.js/lib/parsers/native_formats";
+import Button from "@mui/material/Button";
+import Dialog from "@mui/material/Dialog";
+import DialogActions from "@mui/material/DialogActions";
+import DialogContent from "@mui/material/DialogContent";
+import DialogTitle from "@mui/material/DialogTitle";
```

Comment:
To ommit code duplication we already created generic component with defined DialogActions, DialogContent and DialogTitle: https://github.com/Exabyte-io/exabyte-web-app/blob/dev/src/application/imports/core_cove/components/material-ui/Dialog.tsx. Please use it here as well (button labels are configurable, have a look on other this component usages, like this: https://github.com/Exabyte-io/exabyte-web-app/blob/dev/src/application/imports/science_meve/components/pseudopotential/PseudoUploadDialog.tsx).

Reply:
Links lead to page not found for me